tox.ini

```diff
@@ -1,16 +1,23 @@
 [tox]
-envlist = lint, test
```

Comment:
Will it block the CI flow to put pylint here? These get executed when you run `tox` by itself. Might be better to invoke pylint with `tox -e pylint`